src/vulkan-renderer/exceptions/exception.cpp

```diff
@@ -0,0 +1,7 @@
+#include "inexor/vulkan-renderer/exceptions/exception.hpp"
+
+namespace inexor::vulkan_renderer::exceptions {
+
+Exception::Exception(std::string message) : std::runtime_error(message) {}
+
+}; // namespace inexor::vulkan_renderer::exceptions
```

Comment:
```suggestion
```

Not required.

Reply:
So should I delete the entire file?

Reply:
I deleted `exceptions.cpp` and removed this entry.

Reply:
Done.